numpy/core/fromnumeric.py

```diff
@@ -2282,11 +2298,28 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
     >>> np.nanmax(b)
     4.0
 
+    You can use an initializer to compute the maximum of an empty slice, or
+    to initialize it to a different value:
+
+    >>> np.max([], initializer=-np.inf)
+    -inf
+    >>> np.max([50], initializer=25)
```

Comment:
Maybe use something a bit more likely here, which shows that the initializer is just included in the comparison. E.g.,
```
np.max([[-50], [10]], axis=-1, initializer=0)
[[0],
 [10]]
```
(guessing at formatting)